tap-gui/plugins/scc-tap-gui.md

```diff
@@ -6,7 +6,10 @@ The Supply Chain Choreographer (SCC) plug-in enables you to visualize the execut
 
 ## <a id="prerequisites"></a> Prerequisites
 
-You must have either of the Full or View profiles installed on your cluster, which includes the Tanzu Application Platform GUI.
+You must have installed: 
+- Tanzu Application Platform GUI
+- At least one of the OOTB supply chains
+- Copied the secret (work with James R on wording) for the metadata store (if using a supply chain that includes scanning) following the instructions here <put link to instructions on how to copy the secret into the tap-values.yaml file. 
```

Comment:
Hi @Superv0n, Thanks for the PR. Could you please kindly confirm the instructions in line 12 is completed? For example:

- (work with James R on wording)
- <put link to instructions on how to copy the secret into the tap-values.yaml file

Thanks!

Reply:
thank you, these instructions are no longer required and I've submitted a new pr as well.

Reply:
Thanks @Superv0n. I'm closing this story now.